packages/shared/src/components/cards/PostCard.tsx

```diff
@@ -133,7 +131,7 @@ export const PostCard = forwardRef(function PostCard(
             className={classNames(
               'rounded-b-12',
               insaneMode
-                ? 'relative tablet:absolute tablet:right-0 tablet:bottom-0 tablet:left-0 mt-2 tablet:mt-0 tablet:border-0 border-t border-theme-divider-tertiary'
+                ? 'relative tablet:absolute tablet:right-0 tablet:bottom-0 tablet:left-0 mt-2.5 tablet:mt-0 tablet:border-0 border-t border-theme-divider-tertiary'
```

Comment:
I think the value should be divisible by 4 as part of our guideline. Also,  the design team has mentioned as well that they've made mistake on the margins where it has an excess of 2px. From 24px to 26px. Probably they have missed updating this one.

```suggestion
                ? 'relative tablet:absolute tablet:right-0 tablet:bottom-0 tablet:left-0 mt-2 tablet:mt-0 tablet:border-0 border-t border-theme-divider-tertiary'
```

Reply:
Good point, i'll check they are still updating some other bugs they found, will take this into account as well.